Prac_06/guitar.py

```diff
@@ -0,0 +1,25 @@
+"""
+Started: 3:00
+Finished: 3:10
+Took 10 minutes
+"""
+
+
+class Guitar:
+    """Make a guitar processing class"""
+
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return f"{self.name} ({self.year}) : ${self.cost:,.2f}"
+
+    def get_age(self):
+        age = 2022 - int(self.year)
+        return age
+
+    def is_vintage(self):
+        if self.get_age() >= 50:
+            return True
```

Comment:
Will this return something if the condition is not met? (Im not sure) but I do think it would be better practice to have an else statement.